CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ project adheres to
 
 ## [Unreleased]
 
-## [0.9.0]
+## [0.9.0-rc1]
```

Comment:
When you wrap this in square brackets and don't add a link after you need to make sure there is a defined link defined elsewhere. i.e. check near the bottom for examples, create a link `[0.9.0-rc1]: <URL>`

Reply:
Yep, sorry for missing that! Fixed this in the `CHANGELOG.md` and in the pre-release description. Also fixed a small issue with the `Unreleased` link, could you please re-review?